### PR TITLE
Clarify wording for error conditions

### PR DESCRIPTION
**Pull request by randomairborne**
very basic wording cleanup. Confused a few people.

### CODE REVIEW

File: _help/guides/getting-minecraft-launcher-log/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ The launcher log contains history from when the launder is started up until the
 
 #### Step 1
 
-Start Minecraft and run it to the point where the crash/error occurs. Then close Minecraft and keep it closed.
+Start the launcher and use it to the point where the error occurs. Then close Minecraft and keep it closed.
```
Comment (Greymagic27) [inline _help/guides/getting-minecraft-launcher-log/index.md:24]: I don't think this is clearer. It's stating now to load up the launcher, to "use" it, and then close Minecraft when the error occurs. My issue with this is that the user is just being told to load up the launcher, not being told to launch the game until a crash occurs.

I think the old wording is actually better

**Review by Greymagic27 — CHANGES_REQUESTED** [review]

File: _help/guides/getting-minecraft-launcher-log/index.md (hunk 1 above)
Comment (randomairborne) [reply _help/guides/getting-minecraft-launcher-log/index.md:24]: Not all errors that warrant a launcher log are game crashes, though.

Comment (Greymagic27) [reply _help/guides/getting-minecraft-launcher-log/index.md:24]: But then that leaves the question that will the user understand that they need to load up the game for game crashes, if it just says to load up the launcher

Comment (randomairborne) [reply _help/guides/getting-minecraft-launcher-log/index.md:24]: hmm. this may need bikeshedding

Comment (duendwang) [reply _help/guides/getting-minecraft-launcher-log/index.md:24]: I replied with a comment but it didn't get sent.

Perhaps "Try to start Minecraft and ..."

**Comment by duendwang** [comment]
Better wording would be "try to start minecraft...."

**Pull request closed without merging**